tests/integration/devfile-page/devfile-page.spec.ts

```diff
@@ -0,0 +1,83 @@
+/// <reference types="cypress" />
+/**
+ * page type: stack, sample
+ */
+describe('Test loaded components', () => {
+  beforeEach(() => {
+    cy.viewport(1920, 1080);
+  });
+  it('Stack page', () => {
+    cy.visit('http://localhost:3000/devfiles/Community+java-maven');
+    cy.get("[id='dev-page-header']").should('be.visible');
```

Comment:
`id` should be replace by `data-testid` since its the standard for e2e and unit testing

Reply:
resolved